zboss/README.rst

```diff
@@ -6,3 +6,21 @@ ZBOSS Zigbee stack
 The |NCS|'s Zigbee stack uses ZBOSS – a portable, high-performance Zigbee software protocol stack that allows for interoperability, customizing, testing, and optimizing of your Zigbee solution.
 
 For detailed documentation of the ZBOSS API and instructions on how to use it, see `ZBOSS API documentation`_.
+
+ZBOSS configuration
+===================
+
+The ZBOSS stack is distributed as a set of precompiled linkable libraries.
+There are two variants of the libraries that support different Zigbee device roles, each variant having its own configuration.
+For an overview of the ZBOSS library configuration, see the following files:
```

Comment:
This is not just an overview, but a complete list of configs.

Reply:
@tomchy @greg-fer If changing this triggers CI run I'd prefer to keep it as is.